hips/draw/simple.py

```diff
@@ -178,11 +177,52 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_hips_tile_grid(self) -> None:
+        """Draw lines on the output image (mainly used for debugging).
+
+        A Jupyter notebook showcasing this functionality can be found
+        here: https://github.com/adl1995/HIPS-to-Py/blob/master/examples/Draw_markers_plots_image.ipynb
```

Comment:
Don't link to your notebook from the docstring, just from a Github comment.
External code using `hips` will break very quickly, as we refactor and improve our `hips` package.